app/jobs/build_submission_bundle_job.rb

```diff
@@ -5,7 +5,7 @@ def perform(submission_id)
     begin
       submission.generate_irs_submission_id!
     rescue StandardError => e
-      submission.transition_to!(:failed, error_code: 'IRS-ID-FAIL', raw_response: e.inspect)
+      submission.transition_to!(:failed, error_code: 'IRS-ID-FAIL', raw_response: e.inspect) unless Rails.env.heroku?
```

Comment:
Oh nice! I was also thinking about how to prevent this issue and opted to add transition to reject to the failed options and add a guard to only allow on non-prod environments but I guess this works too!

```
guard_transition(from: :failed, to: :rejected) do |_submission|
    !Rails.env.production?
  end
  ```

Reply:
oh i was just gonna delete this but we could also add something more permanent, is that what you meant?